apps/frontend/src/components/RequestBox/RequestBox.tsx

```diff
@@ -46,18 +55,25 @@ export default function RequestBox({ requests }) {
         Create request
       </button>
       <div className={styles.form}>
-        <SearchFilterForm
-          filterStatus={requestsType}
-          setFilterStatus={setRequestsType}
-          requestSearchValue={requestSearchValue}
-          setRequestSearchValue={setRequestSearchValue}
-        />
+        <Form className={styles.form}>
+          <Form.Group>
+            <Form.Control type="text" placeholder="Search requests by title" value={searchCriteria.searchTerm} 
+            onChange={event => searchRequests((event.target).value)}></Form.Control>
```

Comment:
There's some bug in the searchRequests functionality. Sometimes it doesn't filter when I only input one character, and sometimes it filters but also keeps some requests that don't meet the criteria.

Reply:
found it. 😅 should do `startsWith` instead of `includes` unless that was the desired functionality to resemble keyword search and not an exact title match

Reply:
Yes, I wanted to match based on keywords. I thought that maybe a user only wants to search for issues that have 'cat' in the title for example. What do you think?

Reply:
agreed